src/components/Chat/RoomMenu.tsx

```diff
@@ -118,9 +123,12 @@ export const RoomButtons: React.FC<RoomMenuProps> = ({ onChange, onClose }) => {
       ref={ref}
     >
       <div className="room_menu__header">
-        <button className="room_menu__toggle" onClick={onClose}>
-          <BsArrowBarRight />
-        </button>
+        {!mainActiveTopic && (
+          <button className="room_menu__toggle--on" onClick={handleToggleClick}>
+            <BsArrowBarRight />
+          </button>
+        )}
```

Comment:
this shouldn't be hidden when chat threads are open - hiding this prevents desktop users from closing the drawer